lib/src/multipart_request.dart

```diff
@@ -83,7 +83,7 @@ class MultipartRequest extends BaseRequest {
   ByteStream finalize() {
     // TODO(nweiz): freeze fields and files
     var boundary = _boundaryString();
-    headers['content-type'] = 'multipart/form-data; boundary="$boundary"';
+    headers['content-type'] = 'multipart/form-data; boundary=$boundary';
```

Comment:
I think we'll have to remove 39 (`'`) and 61 (`=`) in `_BOUNDARY_CHARACTERS` for this to really work. I'd actually leave just alpha-numerics, 95 (underscore) and 45 (hyphen). I think all others play some special role in URL encoding, so I'd leave them out just in case.